api/controllers/mobile/app.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * mobile/app.js
+ * @apiDescription Respond with the index.html of the Mobile PWA
+ *
+ * @api {get} /mobile/app/:ID
+ * @apiGroup Mobile
+ * @apiPermission None
+ * @apiUse successRes
+ * @apiSuccess (200) {html}
+ */
+const async = require("async");
+
+// var inputParams = {
+//    tenant: { string: true, optional: true },
+// };
+
+module.exports = function (req, res) {
+   req.ab.log("mobile/app():");
+
+   var user = req.ab.user;
+   let appID = req.ab.param("ID");
+   let tenantID = req.ab.tenantID;
+   if (!tenantID || tenantID == "??") {
```

Comment:
With the changes in #118 we should never get here. Would need to change `authTenant` if we're expecting use a param to resolve the tenant here.